app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
Tìm hiểu và dùng i18n nào em

Reply:
Dùng I18n cho text sample app nè

Cả giá trị text trong link to nữa



"en" thành :en lun